serpapi/code-challenge
# extracting a list of Van Gogh paintings

**Why this PR?**
This PR addresses the need to extract a list of Van Gogh paintings from the application. By implementing this functionality, users will be able to access a comprehensive collection of Van Gogh's artworks.

**What does this PR do?**
This PR introduces a new feature that allows the application to scrape and extract a list of Van Gogh paintings from reliable sources. It includes the necessary code changes to fetch the data, parse it, and transform it into a structured format that can be easily utilized within the application.

**How can this PR be tested?**
To test this PR, perform the following steps:

- Run the application locally `rackup -p 3000`
- Navigate to `/scrape` endpoint
- Run the test as well `rspec` to make sure the solution works against other similar result pages having different.
- Verify that all functionalities related to the Van Gogh paintings extraction are working as expected
